app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        if health > 0:
+            self.alive.append(self)
+
+    def __repr__(self) -> None:
+        return "{"\
+            + f"Name: {self.name}," \
+              f" Health: {self.health}," \
+              f" Hidden: {self.hidden}"\
+            + "}"
```

Comment:
better to use `()` then `\`
```suggestion
        return (
             f"{{Name: {self.name}, "
             f"Health: {self.health}, "
             f"Hidden: {self.hidden}}}"
         )
```